apps/src/StudioApp.js

```diff
@@ -1853,7 +1853,7 @@ StudioApp.prototype.configureDom = function (config) {
   var referenceArea = document.getElementById('reference_area');
   // noInstructionsWhenCollapsed is used in TopInstructions to determine when to use CSPTopInstructions (in which case
   // display videos in the top instructions) or CSFTopInstructions (in which case the videos are appended here).
-  const referenceAreaInTopInstructions = config.noInstructionsWhenCollapsed && (experiments.isEnabled('resources_tab') || experiments.isEnabled('resourcesTab'));
+  const referenceAreaInTopInstructions = config.noInstructionsWhenCollapsed && (experiments.isEnabled('resources_tab') || experiments.isEnabled('resourcesTab') || experiments.isEnabled('additionalResources'));
```

Comment:
Obviously somewhat external to this PR, but I count five places we check
```js
experiments.isEnabled('resources_tab') || experiments.isEnabled('resourcesTab')
```
It's probably worth extracting a helper for this - `experiments.isResourcesTabEnabled()` would be fine.  If you're fine with resolving a merge/rebase, we could even do that in a separate PR before merging this change.

Reply:
In parallel to this PR, I'm working on another one that actually removes all the resourcesTab experiments checks (because we are releasing the resourcesTab to the world). Depending on the review process for both PRs, that one might go out before this one, but probably +/- 1 week of this one, so I don't think adding the helper is much use.